### PR TITLE
Azure DevOps integration

### PR DESCRIPTION
**Pull request by yohanb**
Hi,
as per issue #38, this is my contribution to add the Azure DevOps integration to the helm chart.
Cheers!

### CODE REVIEW

**Review by lkysow — APPROVED** [review]
Looks great! I think just bump up the version and we're good.

File: charts/atlantis/values.yaml [hunk 1]
```diff
@@ -37,6 +37,14 @@ orgWhitelist: <replace-me>
 #   baseURL: https://bitbucket.yourorganization.com
 # (The chart will perform the base64 encoding for you for values that are stored in secrets.)
 
+# If using Azure DevOps, specify like the following:
+# azuredevops:
```
Comment (spirosoik) [inline charts/atlantis/values.yaml:41]: it's better to have a flag like `enabled: false`

**Review by spirosoik — APPROVED** [review]
good job!!!!

File: charts/atlantis/values.yaml (hunk 1 above)
Comment (lkysow) [reply charts/atlantis/values.yaml:41]: yeah that's a better pattern but the other VCS stanzas don't have that so I think this is more consistent.

**Comment by Eugst** [comment]
Can this one will be fixed and merged, please?

**Pull request merged**